### ISSUE TITLE
XML escaping is broken in exported NCIT OWL

### ISSUE THREAD

**Issue by cmungall** [ontodev/ncit-obo#17]
try this on https://build.berkeleybop.org/job/ncit-obo/lastStableBuild/

```
robot convert -i neoplasm.owl -o foo.owl
```

gives:

```
Caused by: org.semanticweb.owlapi.model.OWLOntologyStorageException: java.io.IOException: XML literal is not self contained: "&lt;ncicp:ComplexDefinition&gt;&lt;ncicp:def-definition&gt;Primary or metastatic neoplasms of the CEREBELLUM. Tumors in this location frequently present with ATAXIA or signs of INTRACRANIAL HYPERTENSION due to obstruction of the fourth ventricle. Common primary cerebellar tumors include fibrillary ASTROCYTOMA and cerebellar HEMANGIOBLASTOMA. The cerebellum is a relatively common site for tumor metastases from the lung, breast, and other distant organs. (From Okazaki &amp; Scheithauer, Atlas of Neuropathology, 1988, p86 and p141)&lt;/ncicp:def-definition&gt;&lt;ncicp:def-source&gt;MSH2003_2003_05_12&lt;/ncicp:def-source&gt;&lt;/ncicp:ComplexDefinition&gt;"
        at org.semanticweb.owlapi.rdf.rdfxml.renderer.RDFXMLStorer.storeOntology(RDFXMLStorer.java:61)
        at org.semanticweb.owlapi.util.AbstractOWLStorer.store(AbstractOWLStorer.java:99)
        at org.semanticweb.owlapi.util.AbstractOWLStorer.storeOntology(AbstractOWLStorer.java:64)
        at uk.ac.manchester.cs.owl.owlapi.OWLOntologyManagerImpl.saveOntology(OWLOntologyManagerImpl.java:1264)
        at org.obolibrary.robot.IOHelper.saveOntology(IOHelper.java:443)
        ... 6 more
Caused by: java.io.IOException: XML literal is not self contained: "&lt;ncicp:ComplexDefinition&gt;&lt;ncicp:def-definition&gt;Primary or metastatic neoplasms of the CEREBELLUM. Tumors in this location frequently present with ATAXIA or signs of INTRACRANIAL HYPERTENSION due to obstruction of the fourth ventricle. Common primary cerebellar tumors include fibrillary ASTROCYTOMA and cerebellar HEMANGIOBLASTOMA. The cerebellum is a relatively common site for tumor metastases from the lung, breast, and other distant organs. (From Okazaki &amp; Scheithauer, Atlas of Neuropathology, 1988, p86 and p141)&lt;/ncicp:def-definition&gt;&lt;ncicp:def-source&gt;MSH2003_2003_05_12&lt;/ncicp:def-source&gt;&lt;/ncicp:ComplexDefinition&gt;"
```

**Comment by cmungall** [comment]
- [ ] ComplexDefinitions should always be converted to human-readable literals
- [ ] Any remaining `&lt;` dodginess should be checked for post-processing, with an option to fail-fast or simply remove this if any remain
- [ ] The travis and jenkins jobs should test for roundtrip
- [ ] NCIT technical staff should be informed if anything is not syntactically correct (or the OWLAPI developers should be informed if something is wrong here - I would expect anything written by OWLAPI to be readable by it...)

**Comment by jamesaoverton** [comment]
I think this was caused by #20 and fixed in 31b9f53ff82f80f6a660a3a6acaa04314b29684f.

I added tests to the Jenkins job (currently paused, I guess).
